packages/jumpstarter-driver-flashers/oci_bundles/build_bundle.sh

```
@@ -12,12 +12,16 @@ echo "Building and pushing ${FLASHER_OCI_CONTAINER}"
 set -x
 
 cd "${BUNDLE_FILES}"
+MANIFESTS=
+for file in $(ls -1 *.yaml); do
```

Shellcheck (0.10.0)

[warning] Iterating over ls output is fragile. Use globs.

(SC2045)